### ISSUE TITLE
compilation error: content_context.cpp:85:89: error: ‘class QString’ has no member named ‘toAscii’

### ISSUE THREAD

**Issue by chmorgan** [rectalogic/webvfx#9]
pt -isystem /usr/include/qt5/QtCore -I../build/release/.moc/webvfx -I/usr/lib64/qt5/mkspecs/linux-g++ -o ../build/release/.obj/webvfx/content_context.o content_context.cpp
content_context.cpp: In member function ‘QUrl WebVfx::ContentContext::getImageUrl(const QString&)’:
content_context.cpp:85:89: error: ‘class QString’ has no member named ‘toAscii’
return QUrl::fromEncoded(QString("image://webvfx/%1/%2").arg(name).arg(renderCount).toAscii(), 
^
Makefile.Release:344: recipe for target '../build/release/.obj/webvfx/content_context.o' failed
make[2]: **\* [../build/release/.obj/webvfx/content_context.o] Error 1

**Comment by chmorgan** [comment]
Looks like it might be due to deprecated functions in qt 5.6.

**Comment by rectalogic** [comment]
Try the mlt branch, it's been updated for qt5

**Comment by chmorgan** [comment]
Yep, that fixed it. Thanks for the pointer there. Still struggling to build shotcut, trying both by building things separately and the build script in the source repository but this helps on the separate build approach.
